scripts/core/EXT.rst

```diff
@@ -42,9 +69,9 @@ The following naming conventions must be followed for **standard** extensions:
 
 ## --validate=off
   - All extension functions must be postfixed with `Ext`
-  - All macros must use all caps `${X}_NAME_EXT` convention
-  - All structures, enumerations and other types must follow `${x}_name_ext_t` snake case convention
-  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` convention
+  - All macros must use all caps `${X}_NAME_EXT` or `${S}_NAME_EXT` convention
+  - All structures, enumerations and other types must follow `${x}_name_ext_t` or `${s}_name_ext_t` snake case convention
+  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` or `${S}_ENUM_EXT_ETOR_NAME` convention
```

Comment:
"All enumerator values must use *\<PREFIX\>*_ALL_CAPS convention, where **PREFIX** is appropriately applied for the enumerator namespace, i.e. `${X}_ENUM_EXT_ETOR_NAME`, `${S}_ENUM_EXT_ETOR_NAME` or `${T}_ENUM_EXT_ETOR_NAME`."

Reply:
fixed